plugins/janus_nosip.c

```diff
@@ -786,11 +786,6 @@ int janus_nosip_init(janus_callbacks *callback, const char *config_path) {
 	}
 	JANUS_LOG(LOG_VERB, "Local IP set to %s\n", local_ip);
 
-#ifdef HAVE_SRTP_2
-	/* Init randomizer (for randum numbers in SRTP) */
-	RAND_poll();
-#endif
-
```

Comment:
We should check if removing the randomizers init here (and the SIP plugin) is ok, since plugins are loaded with `dlopen` and we only do the init in the core.

Reply:
Calling RAND_poll() it not necessary at all: https://www.openssl.org/docs/manmaster/man3/RAND_poll.html

>  These functions can be used to seed the random generator and to check its seeded state. In general, manual (re-)seeding of the default OpenSSL random generator (RAND_OpenSSL(3)) is not necessary (but allowed), since it does (re-)seed itself automatically using trusted system entropy sources. This holds unless the default RAND_METHOD has been replaced or OpenSSL was built with automatic reseeding disabled, see RAND(7) for more details.